joda-time/src/main/java/org/joda/time/field/PreciseDateTimeField.java

```diff
@@ -77,13 +77,14 @@ public PreciseDateTimeField(DateTimeFieldType type,
      * @return the amount of fractional units extracted from the input.
      */
     @Override
-    public int get(long instant) {
-        if (instant >= 0) {
-            return (int) ((instant / getUnitMillis()) % iRange);
-        } else {
-            return iRange - 1 + (int) (((instant + 1) / getUnitMillis()) % iRange);
+        public int get(long instant) {
+            if (instant >= 0) {
+                return (int)((instant / getUnitMillis()) % iRange);        
+            } 
+            else {
+                return iRange - 1 + (int) (((instant + 1) / getUnitMillis()) % iRange);
+            }
         }
-    }
```

Comment:
Why does it appear as if you're deleting this line (presumably because of some wrong indentation earlier)?

Reply:
Yes it was due to some of the wrong indentations done earlier, after removing the else and its brackets, this line is now on 85.